src/Avalonia.Animation/AnimationInstance`1.cs

```diff
@@ -48,45 +44,45 @@ public AnimationInstance(Animation animation, Animatable control, Animator<T> an
             _targetControl = control;
             _neutralValue = (T)_targetControl.GetValue(_parent.Property);
 
-            _speedRatio = animation.SpeedRatio;
+            _speedRatioSub = animation.GetObservable(Animation.SpeedRatioProperty)
```

Comment:
Why are you only subscribing to SpeedRatio but not other properties? 
The subscription is also not necessary since you can re-read values in InternalTick.